### PR TITLE
added 700 pairs to swe-eng

### PR DESCRIPTION
**Pull request by nathanchi**
I added 700 pairs to swe-eng.  Verbs were added. 

Output: https://pastebin.com/5F0bVdb5

### CODE REVIEW

**Comment by nathanchi** [comment]
List of Swedish-English words added:
[11:19 swedish-english verbs added.pdf](https://github.com/apertium/apertium-swe-eng/files/2602769/11.19.swedish-english.verbs.added.pdf)

File: apertium-swe-eng.swe-eng.dix [hunk 1]
```diff
@@ -12755,6 +12755,707 @@
...
+      <e><p><l>nafsa</l><r>nibble<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>nafsa</l><r>snap<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>naja</l><r>bind<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>naja</l><r>lash<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>nalla</l><r>lift<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>nappa</l><r>bite<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>nappa</l><r>nibble<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>nappa</l><r>strike<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>nappa</l><r>take<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>nara</l><r>closely<s n="vblex"/></r></p><par n="vblex_"/></e>
```
Comment (unhammer) [inline apertium-swe-eng.swe-eng.dix:12792]: closely is an adverb  (can `nara` mean "move closer to" here? maybe just remove this one)

File: apertium-swe-eng.swe-eng.dix [hunk 2]
```diff
@@ -12755,6 +12755,707 @@
...
+      <e><p><l>sammankalla</l><r>summon<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sammanpressa</l><r>compress<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sammanstråla</l><r>converge<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sammansvetsa</l><r>weld<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sammanträffa</l><r>coincide<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>såpa</l><r>soap<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sära</l><r>separate<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>satsa</l><r>stake<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>satsa</l><r>venture<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>saxa</l><r>quote<s n="vblex"/></r></p><par n="vblex_"/></e>
```
Comment (unhammer) [inline apertium-swe-eng.swe-eng.dix:13013]: as in "this has been cut-and-pasted from another news paper"? (do we have "citera"?)

File: apertium-swe-eng.swe-eng.dix [hunk 3]
```diff
@@ -12755,6 +12755,707 @@
...
+      <e><p><l>sammanträffa</l><r>coincide<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>såpa</l><r>soap<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sära</l><r>separate<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>satsa</l><r>stake<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>satsa</l><r>venture<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>saxa</l><r>quote<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>schacka</l><r>check<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>seeda</l><r>seed<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sela</l><r>gear<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sela</l><r>harness<s n="vblex"/></r></p><par n="vblex_"/></e>
```
Comment (unhammer) [inline apertium-swe-eng.swe-eng.dix:13017]: these look like nouns?

File: apertium-swe-eng.swe-eng.dix [hunk 4]
```diff
@@ -12755,6 +12755,707 @@
...
+      <e><p><l>stycka</l><r>quarter<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sucka</l><r>sigh<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sudda</l><r>erase<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sukta</l><r>hanker<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sukta</l><r>sigh<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sukta</l><r>yearn<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sukta</l><r>crave<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>sumpa</l><r>blow<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>surfa</l><r>surf<s n="vblex"/></r></p><par n="vblex_"/></e> 
+      <e><p><l>surfa</l><r>sailboard<s n="vblex"/></r></p><par n="vblex_"/></e>
```
Comment (unhammer) [inline apertium-swe-eng.swe-eng.dix:13191]: is sailboard a verb?

**Review by unhammer — CHANGES_REQUESTED** [review]
Could you show the output of running the English through too? there's some stuff in there that doesn't seem very verby

File: apertium-swe-eng.swe-eng.dix (hunk 4 above)
Comment (nathanchi) [reply apertium-swe-eng.swe-eng.dix:13191]: I'll fix these issues.

File: apertium-swe-eng.swe-eng.dix (hunk 3 above)
Comment (nathanchi) [reply apertium-swe-eng.swe-eng.dix:13017]: These are all verbs as well as nouns, but I can edit them (or just remove them).

File: apertium-swe-eng.swe-eng.dix (hunk 4 above)
Comment (nathanchi) [reply apertium-swe-eng.swe-eng.dix:13191]: To sailboard is a verb, but I can remove it.

File: apertium-swe-eng.swe-eng.dix (hunk 2 above)
Comment (nathanchi) [reply apertium-swe-eng.swe-eng.dix:13013]: The word citera is in the swe-eng bidix.

**Comment by nathanchi** [comment]
I'm not quite sure what's wrong with the eng-swe direction; when I tried to run an English word to get a Swedish translation, it didn't work.

For example, I ran "angel" through the bidix:
```
NathanChisiMac:apertium-swe-eng nathanchi$ echo "angel" | apertium -d . eng-swe
Warning: Decomposition symbol <:compound:only-L> not found
Warning: Decomposition symbol <:compound:R> not found
```
I got that warning, however, and it didn't work.  

Do you know what the problem might be?

File: apertium-swe-eng.swe-eng.dix (hunk 4 above)
Comment (unhammer) [reply apertium-swe-eng.swe-eng.dix:13191]: nah, if it's used it's probably fine; "surf" is in there for the lrx to default to

File: apertium-swe-eng.swe-eng.dix (hunk 3 above)
Comment (unhammer) [reply apertium-swe-eng.swe-eng.dix:13017]: IIUC, `sela` is used as as in `att sela på hästen`, putting the harness on the horse[1] – I think I would remove those `sela` translations, unless there's a verb that is used for putting on harnesses (the most common use of "to harness" is so different I think it'd lead to some confusing translations)

[1] "Först ska vi sela på hästen!" says https://lycklighast.wordpress.com/2012/03/01/korning-for-dummies-del-3-hur-gor-man/ which google translates as "first we will *sell* the horse" 😂

**Comment by unhammer** [comment]
> I'm not quite sure what's wrong with the eng-swe direction; when I tried to run an English word to get a Swedish translation, it didn't work.
> 
> For example, I ran "angel" through the bidix:
> 
> ```
> NathanChisiMac:apertium-swe-eng nathanchi$ echo "angel" | apertium -d . eng-swe
> Warning: Decomposition symbol <:compound:only-L> not found
> Warning: Decomposition symbol <:compound:R> not found
> ```
> 
> I got that warning, however, and it didn't work.
> 
> Do you know what the problem might be?

The `modes.xml` file uses  `lt-proc -e` which turns on compound decomposition (useful for other Germanic languages that don't put spaces in compounds), but it seems the English monodix doesn't include the special symbols that say when compound decomposition should happen, so it just gives a little warning about that. Check the modes file that it uses the right English monodix, and that it only uses `-e` if that dix has sdef's `compound-R` / `compound-only-L`. It should still work though, so something else must be up too.

**Comment by nathanchi** [comment]
I'm sorry, I'm not quite sure if I understand what you mean.  What should I be checking for here?

**Comment by nathanchi** [comment]
If it isn't the problem with the right English monodix, and if it's something else, should I be checking for something different?

**Comment by unhammer** [comment]
So e.g. https://github.com/apertium/apertium-swe-eng/blob/master/modes.xml#L274 includes the `-e` argument (as do the eng-swe-debug modes) but eng.dix doesn't include the compound sdefs. Remove the `-e` argument from the lt-proc eng-swe.automorf.bin (but not swe→eng!) parts of modes.xml to get rid of the warning.

I don't know if it'll make it analyse correctly or not though

**Comment by nathanchi** [comment]
Okay, I removed the -e arguments from each mention of eng-swe.automorf.bin in apertium-swe-eng's modes.xml, and it looks like the warning is gone.  The word still isn't analyzed correctly, however.

**Comment by nathanchi** [comment]
This is what it looks like now:

`NathanChisiMac:apertium-swe-eng-final nathanchi$ echo "angel" | apertium -d . eng-swe
NathanChisiMac:apertium-swe-eng-final nathanchi$ 
`

Do you have any further suggestions?

**Comment by unhammer** [comment]
What about the debug modes? eng-swe-morph etc. (do `ls modes/eng*` to see all)

ie. `echo angel | apertium -d . eng-swe-morph`

**Comment by nathanchi** [comment]
When I ran `echo angel | apertium -d . eng-swe-morph`:

`NathanChisiMac:apertium-swe-eng-final nathanchi$ echo angel | apertium -d . eng-swe-morph
^angel/angel<n><sg>$^./.<sent>$`

**Comment by nathanchi** [comment]
There was no warning when I ran the debug mode.

**Comment by nathanchi** [comment]
Should I be testing each of these modes (e.g. biltrans, dgen, disam, interchunk, lex, postchunk, pretransfer, tagger, and transfer)?

**Comment by nathanchi** [comment]
Running `echo angel | apertium -d . eng-swe-postchunk` yielded the same two warnings.

**Comment by nathanchi** [comment]
Okay, that warning was removed.  Do you have any other suggestions?

**Comment by nathanchi** [comment]
Running `disam` gave this:

```
apertium-swe-eng-final nathanchi$ echo angel | apertium -d . eng-swe-disam
"<angel>"
	"angel" n sg
"<.>"
	"." sent
```

The other debug modes (besides morph) didn't yield anything, though.

File: apertium-swe-eng.swe-eng.dix (hunk 3 above)
Comment (nathanchi) [reply apertium-swe-eng.swe-eng.dix:13017]: Maybe `to yoke` would be more appropriate than `to harness`.  That's more unambiguous.

**Comment by unhammer** [comment]
tried it on my machine, it seems apertium-tagger is going at 100%, presumably missing an `-x` argument

**Comment by unhammer** [comment]
merge master and try again :)

**Comment by nathanchi** [comment]
Okay, I cloned the updated swe-eng bidix to see whether it would work and ran "angel" through it again.
It still doesn't work, though.  Is there something more I should fix?
`NathanChisiMac:apertium-swe-eng nathanchi$ echo "angel" | apertium -d . eng-swe
            NathanChisiMac:apertium-swe-eng nathanchi$`

**Comment by nathanchi** [comment]
swe-eng still works, but eng-swe doesn't work.

**Comment by nathanchi** [comment]
Actually, it works now!  :)

**Comment by nathanchi** [comment]
New PR: https://github.com/apertium/apertium-swe-eng/pull/5

**Comment by nathanchi** [comment]
Okay, thanks for merging the PR!  This corresponds to the GCI task https://codein.withgoogle.com/dashboard/task-instances/6458867622871040/.  When you have time, could you approve it, please?

**Pull request closed without merging**